### DISCUSSION TITLE
Consideration for removing the space at the end of Upstream icons

### DISCUSSION THREAD

**Discussion by jreilly-lukava** [JanDeDobbeleer/oh-my-posh#4774] [General]
https://github.com/JanDeDobbeleer/oh-my-posh/blob/a8f97f5aa2487fdfd25253a83aafae04aadb089c/src/segments/git.go#L495-L500
Just my opinion I guess, but including the space by default seems counter intuitive.  I think it should just be the icon and then the user can add a space if they want one.  This looks especially odd when using it in a url with a terminal that highlights them.

Looks bad IMO:
![image](https://github.com/JanDeDobbeleer/oh-my-posh/assets/30353736/c7e789d7-9dcb-453f-ac88-e26b7cac5add)

Looks good:
![image](https://github.com/JanDeDobbeleer/oh-my-posh/assets/30353736/c8f7f30b-edca-4331-ab3e-23862c17178d)

**Comment by JanDeDobbeleer** [comment]
@jreilly-lukava fair point. Mind creating a PR for this?

**Reply by jreilly-lukava** [reply]
Done.
